api/helpers/auth.js

```diff
@@ -133,8 +133,8 @@ exports.issueToken = function (user,
 
 var hashPassword = function(user, password) {
```

Comment:
I'm not sure how much of this file is being used. I don't think we're actually hashing any passwords, for example. But in the interest of resolving the break in Openshift quickly, we'll get this moving. Thanks for the fix